hack/tests/scaffolding/scaffold-memcached.go

```diff
@@ -77,15 +77,12 @@ func main() {
 		log.Fatalf("Failed to change to %s directory: (%v)", operatorName, err)
 	}
 
-	replace := getGoModReplace(localSDKPath)
```

Comment:
Hi @estroz, 

Could you help me understand it better? Here currently we download the PR  and ensuring that the Memcached mock data will be created with. However, it is not required since the Travis will already install the PR to then call the tests. So, the goal of it is to use SDK already cloned locally by Travis instead of doing all these steps. Am I right?

If yes, its is /lgtm and /approved for me..

Reply:
That is correct.